tests/manual-test-cases/Group19-DRS-Disabled/19-3-DRS-Disabled-Placement.robot

```diff
@@ -0,0 +1,94 @@
+# Copyright 2016-2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 19-3 - DRS-Disabled-Placement
+Resource  ../../resources/Util.robot
+Suite Setup  Wait Until Keyword Succeeds  10x  10m  ROBO SKU Setup
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Keywords ***
+Get Host IP
+    [Arguments]  ${name}
+    Log To Console  Getting host IP for ${name}
+    ${rc}  ${ip}=  Run And Return Rc And Output  govc vm.info ${name} |grep Host |awk '{print $2}'
+    Should Be Equal As Integers  ${rc}  0
+    [Return]  ${ip}
+
+Deploy Stress Container And Return Host IP
+    [Arguments]  ${name}
+    Log To Console  Deploying stress container ${name}...
+    ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name ${name} progrium/stress --vm 5 --vm-bytes 256M --vm-hang 0
+    Should Be Equal As Integers  ${rc}  0
+
+    ${vm_name}=  Get VM display name  ${id}
+    ${host_ip}=  Get Host IP  ${vm_name}
```

Comment:
I think you really just want this:
https://github.com/vmware/vic/blob/master/tests/resources/Docker-Util.robot#L50

instead of the hoops to do this OOB

Reply:
No, I actually need the IP of the host that the vm is running on, but I'll use `Get VM Host Name` instead of this.